skycoin/skywire
# Fix transport manager hangs and various improvements.

Fixes #375

 Changes:	
* Restructured (transport.Manager).SaveTransport to improve readability and ensure we always close the managed transport before calling mTp.wg.Wait(). In fact, we are now using mTp.Close() directly (which does both).
* Visor's RPC client timeout is now optional. Setting a value of 0 duration will result in no timeout.
* Disabled visor RPC client timeout for visor CLI.

Additional Changes:
* Wait for dmsg network during visor startup.

How to test this PR:
* Start a `skywire-visor`.
* Add transports of different types via `skywire-cli`.
* Nothing should hang indefinitely.
